kong/tools/utils.lua

```diff
@@ -1295,6 +1296,36 @@ do
 ]],
   }
 
+  local ngx_log = ngx.log
+  local ERR     = ngx.ERR
+  local custom_error_templates = setmetatable({}, {
+    __index = function(self, format)
+      local template_path_k = "error_template_" .. format
+      local template_path = kong.configuration[template_path_k]
```

Comment:
PDK tests are failing because those tests only initialize the kong global (and contents of it) on demand. See [this](https://github.com/Kong/kong/blob/a9b3f5deff0b51897091dcbea4d4eeb1d77be73c/t/01-pdk/05-client/08-get_protocol.t#L269) for example.

Reply:
thanks @gszr! I had left the tests running and didn't notice the failure yet